src/javascript/JContent/ContentNavigation/NavigationHeader/SwitchersLayout/LanguageSwitcher.jsx

```diff
@@ -33,7 +33,7 @@ export const LanguageSwitcher = ({
     }
 
     if (loading) {
-        return <ProgressOverlay/>;
+        return false;
```

Comment:
The original requirement was to display a disabled dropdown - I don't know if it's possible, but you can check if previous data is available ( siteInfo should contain the previous data ), and use it to display a dropdown , disabled during loading.
If not possible, please update the ticket